LayoutTestBase/Swift/LYTDataValues+Swift.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LYTDataValues+Swift.swift
+//  LayoutTest
+//
+//  Created by Peter Livesey on 9/30/16.
+//  Copyright © 2016 LinkedIn. All rights reserved.
+//
+
+import Foundation
+
+extension LYTDataValues {
+    /**
+     Returns LYTDataValues with these specific values and overrides any subclass's behavior.
+     This function converts all nil values to NSNull which is what the library expects.
+     These will be represented as empty entries in the dictionary.
+     */
+    public convenience init(values: [Any?]) {
```

Comment:
What is the difference between this function signature and the converted version of `- (instancetype)initWithValues:(NSArray *)values`?